modules/backend/ServiceProvider.php

```diff
@@ -104,6 +105,10 @@ protected function registerAssetBundles()
                 $combiner->registerBundle('~/modules/backend/formwidgets/richeditor/assets/js/build.js');
             }
         });
+
+        MixAssets::instance()->registerCallback(function ($mix) {
+            $mix->registerPackage('module-backend.codeeditor', '~/modules/backend/formwidgets/codeeditor/assets/winter.mix.js');
```

Comment:
```suggestion
            $mix->registerPackage('formwidget.codeeditor', '~/modules/backend/formwidgets/codeeditor/assets/winter.mix.js');
```

Reply:
The reason I named it that way is that I see the code editor as a "subset" of the Backend module. I was thinking of introducing a way to namespace packages (like this) and add an argument to `mix:compile` to compile everything within a namespace.

Reply:
Sure, that sounds fine. Could we do `module.backend.formwidgets.codeeditor` perhaps?

Reply:
It'd be `module-backend.formwidgets.codeeditor`, but yep, that works for me.